config/locales/display/display-es.yml

```diff
@@ -45,6 +45,7 @@ es:
     multi_page:
       categories: Categorías
       describe: Crear metadatos
+      pages_need_completion: Páginas que necesitan completarse
```

Comment:
I suspect that this should be something like "Necessitan Completión" to make it parallel with the other Spanish.  (We should check this, however -- my Spanish is pretty bad.)